guidelines/index.html

```diff
@@ -2540,16 +2540,16 @@ <h1 id="title" class="title">Web Content Accessibility Guidelines (WCAG) 2.2</h1
 </dd>
 
             
-            <dt class="new"><dfn data-lt="cognitive function test|Cognitive function test" id="dfn-cognitive-function-test" tabindex="0" aria-haspopup="dialog" data-dfn-type="dfn">Cognitive function test</dfn></dt>
+            <dt class="new"><dfn data-lt="cognitive function test|Cognitive function test" id="dfn-cognitive-function-test" tabindex="0" aria-haspopup="dialog" data-dfn-type="dfn">認知機能テスト</dfn></dt>
 <dd class="new"><p class="change">[New]</p>
    					
-   <p>A task that requires the user to remember, manipulate, or transcribe information. Examples include, but are not limited to:</p>
+   <p>利用者が情報を記憶、操作又は書き写す必要のあるタスク。例には、次のものが含まれるが、これらに限定されない:</p>
    <ul>
-       <li>memorization, such as remembering a username, password, set of characters, images, or patterns. The common identifiers name, e-mail, and phone number are not considered cognitive function tests as they are personal to the user and consistent across Web sites;</li>
-       <li>transcription, such as typing in characters;</li>
-       <li>use of correct spelling;</li>
-       <li>performance of calculations;</li>
-       <li>solving of puzzles.</li>
+       <li>記憶。ユーザー名、パスワード、文字の一式、画像、パターンを覚えること。一般的な識別子である名前、電子メール及び電話番号は、利用者にとって個人的なものであって、ウェブサイト間で一貫しているため、認知機能テストとはみなされない。</li>
+       <li>文字入力などの書き写し。</li>
+       <li>正しい綴りの使用。</li>
+       <li>計算の実行。</li>
+       <li>パズルを解くこと。</li>
```

Comment:
2番目の項目ですが、ここの typing はそのまま「タイピング」で良いのではないでしょうか。また、今の表現だと「入力された文字の書き写し」とも取れる余地がありそうです。上でtranscribeを「転記」としてはどうかというコメントをしましたが、それを活かすなら「文字のタイピングなどによる転記」でしょうか。

とはいえこれはこれで「文字の転記」とも読めるので (実際には「文字の」はタイピングにのみかかる)、原文の表現に合わせて「転記、たとえば文字をタイピングするなど」とするのが良いかもしれません。

最後の項目だけ「……こと」となっているので、どうせなら全部「こと」で終わらせてはどうでしょうか。

また、「正しい綴りの使用」という表現はちょっと何を言っているのか伝わりにくい感があります。ユーザーに課すタスクの話をしているので、「正しい綴りを使わせること」のようにすると伝わりやすいでしょうか。そして仮にそうするなら他の項目も全部「させること」で揃えた方がいいですね。

もろもろ反映するとこうなりますがどうでしょうか。正直なところ迷走感もあり、多分に好みの問題もあろうと思いますが、わかりやすくはなっているように思います。忌憚ないご意見をお聞きできればと……。

```suggestion
       <li>記憶させること、たとえばユーザー名、パスワード、文字の一式、画像、パターンを覚えさせるなど。一般的な識別子である名前、電子メール及び電話番号は、利用者にとって個人的なものであって、ウェブサイト間で一貫しているため、認知機能テストとはみなされない。</li>
       <li>転記させること、たとえば文字をタイピングさせるなど。</li>
       <li>正しい綴りを使わせること。</li>
       <li>計算を実行させること。</li>
       <li>パズルを解かせること。</li>
```